megabus/src/main/java/com/bazaarvoice/megabus/MegabusSource.java

```diff
@@ -0,0 +1,17 @@
+package com.bazaarvoice.megabus;
+
+import com.bazaarvoice.megabus.resource.Coordinate;
+
+import java.util.Iterator;
+
+/**
+ * Defines the Interface for all ad-hoc Megabus Operations.
+ */
+public interface MegabusSource {
+
+    void touch(String table, String key);
```

Comment:
for consistency, it also makes sense to use `Coordinate` class

Reply:
yeah, i too thought about passing coordinate to here from resource. Ok, will do 👍